composer.json

```diff
@@ -81,6 +82,7 @@
         "phpstan/phpstan-symfony": "^0.12.10",
         "psalm/plugin-symfony": "^2.0",
         "psr/event-dispatcher": "^1.0",
+        "sensio/framework-extra-bundle": "^6.1",
```

Comment:
When adding a new `dev` dependency that can be used in production (no test dependency), you should add this to the conflict section.